fission/package.json

```
@@ -1,73 +1,73 @@
-    "format": "npm run prettier:fix && npm run lint:fix",
-    "build:prod": "tsc && vite build --base=/fission/",
-    "assetpack": "run-script-os",
-    "assetpack:nix": "curl -o public/assetpack.zip https://synthesis.autodesk.com/Downloadables/assetpack.zip && unzip -o public/assetpack.zip -d public/",
-    "assetpack:win32": "echo todo"
-  },
-  "dependencies": {
-    "@barclah/jolt-physics": "^0.19.3",
-    "@mui/base": "^5.0.0-beta.40",
-    "@mui/icons-material": "^5.15.20",
-    "@mui/system": "^5.15.20",
-    "@react-three/drei": "^9.96.5",
-    "@react-three/fiber": "^8.15.15",
-    "@vitest/coverage-v8": "^1.6.0",
-    "async-mutex": "^0.5.0",
-    "colord": "^2.9.3",
-    "framer-motion": "^10.13.1",
-    "react": "^18.2.0",
-    "react-colorful": "^5.6.1",
-    "react-dom": "^18.2.0",
-    "react-icons": "^4.10.1",
-    "run-script-os": "^1.1.6",
-    "three": "^0.159.0",
-    "typescript-cookie": "^1.0.6"
-  },
-  "devDependencies": {
-    "@emotion/react": "^11.11.3",
-    "@emotion/styled": "^11.11.0",
-    "@mui/material": "^5.15.6",
-    "@types/node": "^20.4.4",
-    "@types/pako": "^2.0.3",
-    "@types/react": "^18.2.47",
-    "@types/react-dom": "^18.2.18",
-    "@types/three": "^0.160.0",
-    "@typescript-eslint/eslint-plugin": "^7.0.2",
-    "@typescript-eslint/parser": "^7.0.2",
-    "@vitejs/plugin-react": "^4.0.3",
-    "@vitejs/plugin-react-swc": "^3.5.0",
-    "autoprefixer": "^10.4.14",
-    "cssnano": "^6.0.1",
-    "eslint": "^8.56.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-import-resolver-alias": "^1.1.2",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-react-refresh": "^0.4.5",
-    "jsdom": "^24.1.0",
-    "pako": "^2.1.0",
-    "postcss": "^8.4.26",
-    "prettier": "^3.0.0",
-    "protobufjs": "^7.2.6",
-    "protobufjs-cli": "^1.1.2",
-    "tailwindcss": "^3.3.3",
-    "tsconfig-paths": "^4.2.0",
-    "typescript": "^5.2.2",
-    "vite": "^5.1.4",
-    "vite-plugin-singlefile": "^0.13.5",
-    "vitest": "^1.3.1"
-  }
+    "name": "synthesis-fission",
+    "private": false,
+    "version": "0.0.1",
+    "type": "module",
+    "scripts": {
+        "dev": "vite",
+        "build": "tsc && vite build",
+        "lint": "eslint . --ext ts,tsx --report-unused-disable-directives",
+        "preview": "vite preview --base=/fission/",
+        "test": "vitest",
+        "lint:fix": "npm run lint --fix",
+        "prettier": "npx prettier src --check",
+        "prettier:fix": "npm run prettier --write",
+        "format": "npm run prettier:fix && npm run lint:fix",
+        "build:prod": "tsc && vite build --base=/fission/",
+        "assetpack": "run-script-os",
+        "assetpack:nix": "curl -o public/assetpack.zip https://synthesis.autodesk.com/Downloadables/assetpack.zip && unzip -o public/assetpack.zip -d public/",
+        "assetpack:win32": "curl -o public/assetpack.zip https://synthesis.autodesk.com/Downloadables/assetpack.zip && tar -xf public/assetpack.zip"
```

```suggestion
        "assetpack:win32": "curl -o public/assetpack.zip https://synthesis.autodesk.com/Downloadables/assetpack.zip && tar -xf public/assetpack.zip -C public/"
```

Should be placed into the `public/` dir after unzip.